z3/src/context.rs

```diff
@@ -94,6 +94,9 @@ impl Context {
         Symbol::from_string(self, s)
     }
 
+/* Deprecate in favor of the typed variants below?
+    Or even better, the {Bool,Int,etc}Ast::new_const() and ::fresh_const() methods?
+
```

Comment:
this question about deprecating the untyped methods on `Context`

Reply:
Leave and then think about it later.

Reply:
If we leave these in, should they just return `Dynamic` for now?

Reply:
Oh, right, these are commented out. Removing them is fine then.

Reply:
Removed in the latest push